top-commands.go

```diff
@@ -32,6 +32,7 @@ import (
 // ID, type(read or write) and time stamp.
 type LockEntry struct {
 	Timestamp  time.Time `json:"time"`       // When the lock was first granted
+	Elapsed    string    `json:"elapsed"`    // Humanized duration for which lock has been held
```

Comment:
Can this not be done on the `mc` side? such as `time.Since(lk.Timestamp)` ?  - we can then humanize it. Adding a string representation from the server is not necessary and an overkill for a data structure IMO.

Reply:
Here are the reasons why it makes sense for the server to compute duration for which locks were held
- top locks API response should be self contained
- while elapsed time can be derived at client side, computing it at the server-side makes it relative to server system time as it should be.

Reply:
I am against stringifying in this manner and sending humanized forms in a data structure. That is not what Elapsed is meant for if you want to compute we should keep it as `time.Duration` instead.